product_warranty/views/product_warranty.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- return instructions tree view -->
+    <record model="ir.ui.view" id="product_return_instructions_tree_view">
+        <field name="name">product.return.instructions.tree</field>
+        <field name="model">return.instruction</field>
+        <field name="arch" type="xml">
+            <tree string="Return instructions">
+                <field name="name"/>
+                <field name="instructions"/>
+                <field name="is_default"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- return instructions form view -->
+    <record model="ir.ui.view" id="product_return_instructions_form_view">
```

Comment:
just aesthetic change:
id before model

id pattern:
For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...